examples/cra-kitchen-sink/src/components/LifecycleLogger.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+function log(name) {
+  console.log(`LifecycleLogger: ${name}`);
```

Comment:
Feels like a linter warning ?

Reply:
Do you think I should use addon-actions for this instead?

Reply:
You can just ignore the warning =)

Reply:
You should use `@storybook/client-logger`

Reply:
Done.